src/providers/activeLearning/objectDetection.ts

```diff
@@ -0,0 +1,195 @@
+import axios from "axios";
+import * as tf from "@tensorflow/tfjs";
+import { ElectronProxyHandler } from "./electronProxyHandler";
+import { LocalFileSystemProxy, ILocalFileSystemProxyOptions } from "../../providers/storage/localFileSystemProxy";
+
+// tslint:disable-next-line:interface-name
+export interface DetectedObject {
```

Comment:
Why are you not prefixing interface with `I` ?

Reply:
Good point.  It was like that in the original Google sample code.  From my point of view I think this is correct as this is not properly an Interface in the terms that this TS Interface is no where implemented in the code.  It is just a JS Object with no methods.  I think it's a kind of misleading of the TypeScript Interface feature.

Reply:
Should this be eventually a TS Type vs an Interface ?